src/main/java/org/commcare/formplayer/application/WebAppContext.java

```diff
@@ -244,6 +245,10 @@ public ArchiveFileRoot formplayerArchiveFileRoot() {
     }
 
     @Bean
+    public CaseSearchHelper caseSearchHelper() {
+        return new CaseSearchHelper();
+    }
```

Comment:
you could just tag the class with `@Component` and let the auto-discovery pick it up